deployment/single-server/vars/open-zaak.example.yml

```diff
@@ -0,0 +1,18 @@
+---
+
+openzaak_domain: open-zaak.gemeente.lan
+openzaak_ssl: yes
+
+openzaak_db_username: open-zaak
+openzaak_db_password: change-me
+openzaak_secret_key: 'generate-me'  # use https://miniwebtool.com/django-secret-key-generator/
+
+openzaak_version: 0.1.0
```

Comment:
Added.

Note that #429 will move these variables into the playbook to fit the upgrade procedure based on `bumpversion`. It will still be possible to override them though.